test/rest-api/response-patching.mocks.ts

```diff
@@ -48,6 +48,21 @@ const worker = setupWorker(
       }),
     )
   }),
+
+  rest.get('/posts', async (req, res, ctx) => {
+    const originalResponse = await ctx.fetch(req)
+
+    return res(
+      ctx.json({
+        ...originalResponse,
+        mocked: true,
+      }),
+    )
+  }),
+
+  rest.head('/posts', async (req, res, ctx) => {
+    return res(ctx.json(null))
```

Comment:
I would be careful with using `null` as the response body, as it may produce a false positive test result given there is unintentionally no body. Please, could you set a certain text/JSON body in this response resolver? So we are sure we are asserting the mocked response body in the test. Thanks.